src/PrestaShopBundle/Form/Admin/Sell/Product/SEO/SEOType.php

```diff
@@ -243,11 +245,15 @@ private function getFriendlyAlterMessages(): array
     public function configureOptions(OptionsResolver $resolver)
     {
         parent::configureOptions($resolver);
-        $resolver->setDefaults([
-            'label' => $this->trans('Search Engine Optimization', 'Admin.Catalog.Feature'),
-            'label_tag_name' => 'h2',
-            'label_subtitle' => $this->trans('Improve your ranking and how your product page will appear in search engines results.', 'Admin.Catalog.Feature'),
-            'required' => false,
-        ]);
+        $resolver
+            ->setDefaults([
+                'product_id' => null,
+                'label' => $this->trans('Search Engine Optimization', 'Admin.Catalog.Feature'),
```

Comment:
That's not the scope of this PR but I modified it regardless
Although I'm surprised, since SEO is an acronym shouldn't we use capital for each word of the acronym? `SEO = Search Engin Optimization` ?